app/Services/PostService.php

```diff
@@ -73,6 +81,12 @@ public function update($signup, $data)
             info('post_created', ['id' => $postOrSignup->id, 'signup_id' => $postOrSignup->signup_id]);
         }
 
+        // Dispatch job to send signup to Quasar
+        // is this ever a signup?
+        if (config('features.pushToQuasar') && $postOrSignup instanceof Post) {
+            SendPostToQuasar::dispatch($postOrSignup);
+        }
+
```

Comment:
@katiecrane I think we would want to do this when the signup's `why_participated` (and `quantity` before we switch) is updated?

Reply:
I'll add a check to see if it's a Signup and if so, trigger the Signup job 👍